packages/protocol/test/authorizers/Api3TokenLock.sol.js

```diff
@@ -0,0 +1,755 @@
+            expect(
+              await api3TokenLock.airnodeToRequesterToBlockStatus(
+                hre.ethers.constants.AddressZero,
+                roles.requester.address
+              )
+            ).to.equal(true);
+
+            await expect(
+              api3TokenLock.connect(roles.sponsor).lock(airnodeAddress, roles.requester.address)
+            ).to.revertedWith(Errors.RequesterBlocked);
+
+            await expect(
+              api3TokenLock.connect(roles.sponsor).lock(roles.unknown.address, roles.requester.address)
+            ).to.revertedWith(Errors.RequesterBlocked);
+          });
+
+          it('emits event', async () => {
+            await expect(api3TokenLock.connect(roles.metaAdmin).blockRequester(airnodeAddress, roles.requester.address))
+              .to.emit(api3TokenLock, 'BlockedRequester')
+              .withArgs(airnodeAddress, roles.requester.address, roles.metaAdmin.address);
+          });
+        });
+      });
+
+      context('caller is superAdmin', async () => {
+        context('requester is blocking on an airnode', async () => {
+          it('sets the block', async () => {
+            await api3TokenLock.connect(roles.superAdmin).blockRequester(airnodeAddress, roles.requester.address);
+
+            expect(
+              await api3TokenLock.airnodeToRequesterToBlockStatus(airnodeAddress, roles.requester.address)
+            ).to.equal(true);
+
+            await expect(
+              api3TokenLock.connect(roles.sponsor).lock(airnodeAddress, roles.requester.address)
+            ).to.revertedWith(Errors.RequesterBlocked);
+
+            await api3TokenLock.connect(roles.sponsor).lock(roles.unknown.address, roles.requester.address);
+            const now = (await hre.ethers.provider.getBlock(await hre.ethers.provider.getBlockNumber())).timestamp;
+            const expectedUnlockTime = now + ONE_MINUTE_IN_SECONDS;
+
+            expect(
+              await api3TokenLock.sponsorLockAmount(
+                roles.unknown.address,
+                roles.requester.address,
+                roles.sponsor.address
+              )
+            ).to.equal(LOCK_AMOUNT);
+            expect(
+              await api3TokenLock.sponsorUnlockTime(
+                roles.unknown.address,
+                roles.requester.address,
+                roles.sponsor.address
+              )
+            ).to.equal(expectedUnlockTime);
+          });
+
+          it('emits event', async () => {
+            await expect(
+              api3TokenLock.connect(roles.superAdmin).blockRequester(airnodeAddress, roles.requester.address)
+            )
+              .to.emit(api3TokenLock, 'BlockedRequester')
+              .withArgs(airnodeAddress, roles.requester.address, roles.superAdmin.address);
+          });
+        });
+        context('requester is being blocked globally', async () => {
+          it('sets the block', async () => {
+            await api3TokenLock
+              .connect(roles.metaAdmin)
```

Comment:
Shouldn't this be the superAdmin instead?

Reply:
oops, yes my bad. Fixed in the new commit